boards/arm/sam_e70_xplained/sam_e70_xplained_defconfig

```diff
@@ -11,6 +11,7 @@ CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_USART_SAM=y
+CONFIG_GPIO=y
```

Comment:
#41550 should solve GPIO dependency.

Reply:
I'll remove this line and rebase when your PR is merged